addons/compat_sog/CfgMagazines/belts.hpp

```diff
@@ -12,3 +12,8 @@ class vn_m16_mag_base;
 class vn_m63a_100_mag: vn_m16_mag_base {
     ACE_isBelt = 1;
 };
+class vn_mg42_50_mag: vn_lmgmag_base {
+    ACE_isBelt = 1;
+};
+class vn_mg42_50_t_mag: vn_mg42_50_mag {};
+class vn_rpd_125_mag: vn_rpd_100_mag {};
```

Comment:
```suggestion
```
ok to drop these imho

Reply:
True. Not sure what I was thinking when I added them. Purposes of explicitness, I suppose.